src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier() {
+        this.random = new Random();
+    }
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Please see checklist:
https://github.com/SukhachovS/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants